src/Tests/Mapping/Metafields/LocalMetadataTests.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Collections.Generic;
+using FluentAssertions;
+using Nest;
+using Tests.Mapping.Types.Core.Text;
+using Xunit;
+
+namespace Tests.Mapping.Metafields
+{
+    public class LocalMetadataTests
+	{
+		[Fact]
```

Comment:
@russcam I caught this in https://github.com/elastic/elasticsearch-net/commit/c8cc85fa5b0b57bcaac72ea4be48b0c401f384f5